concourse/model/traits/update_component_deps.py

```diff
@@ -159,6 +159,29 @@ def merge_method(self):
         doc='whether or not created PRs should be automatically merged. **deprecated**',
         type=MergePolicy,
     ),
+    AttributeSpec.optional(
+        name='additional_ctx_repositories',
+        default=dict(),
+        doc='''
+            Additional context repositories to look up components in. The entries are expected
+            to be mappings of context repo config names to a list of component names.
+
+            Example:
+
+            .. code-block:: yaml
+
+                additional_ctx_repositories:
+                    ctx_repo_config_name:
+                    - example/component/name
+                    - another/example/component
+                    ctx_repo_config_name_two:
+                    - yet/another/component
+
+            If a component name is configured for two context repository config names the last
```

Comment:
see above. we should rather specify component-name-prefixes, rather than full component-names.